src/io/flutter/run/daemon/FlutterApp.java

```diff
@@ -234,6 +234,10 @@ public void processTerminated(@NotNull ProcessEvent event) {
         // Send analytics about whether this session used the reload workflow, the restart workflow, or neither.
         final String workflowType = app.reloadCount > 0 ? "reload" : (app.restartCount > 0 ? "restart" : "none");
         FlutterInitializer.getAnalytics().sendEvent("workflow", workflowType);
+
+        // Send the ratio of reloads to restarts.
+        final double reloadRestartRatio = app.reloadCount / (app.restartCount + 1);
```

Comment:
wouldn't it be better to send 
app.reloadCount / (app.reloadCount + app.restartCount)
and just call it reloadFraction?
I am assuming the +1 in restartCount is there just to avoid dividing by zero.

Reply:
Multiplying by 100 (and getting two additional decimal points) sounds good.

The 1 is there partly to avoid divide by zero, and partly counting the process termination as a restart.

I'm not sure that measuring as a reload fraction gains us anything over using a reload ratio; @InMatrix, thoughts?

Reply:
Either is fine. Reload fraction might have a slight advantage of being easier to talk about than reload/restart ratio?

Reply:
I changed this to reloadFraction (reload / (reload + restart)), and send in the value multiplied by 100 (since we can only send ints. So, 100 == all reload, and 0 == no reloads.